### DISCUSSION TITLE
PaddleX/latest/pipeline_usage/tutorials/cv_pipelines/image_anomaly_detection

### DISCUSSION THREAD

**Discussion by giscus** [PaddlePaddle/PaddleX#2707] [General]
# PaddleX/latest/pipeline_usage/tutorials/cv_pipelines/image_anomaly_detection

https://paddlepaddle.github.io/PaddleX/latest/pipeline_usage/tutorials/cv_pipelines/image_anomaly_detection.html

**Comment by SolMing** [comment]
API 接口地址文档给错了，访问一直 404 ，后面通过代码找到的地址：
image-anomaly-detection

**Reply by cuicheng01** [reply]
您好，最新的已经修复，请尝试一下呢
